test/hydro/volume_of_the_submerged_body_part_from_body_density_and_liquid_density_test.py

```diff
@@ -0,0 +1,51 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+
+from symplyphysics.laws.hydro import volume_of_the_submerged_body_part_from_body_density_and_liquid_density as submerged
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    body_volume = Quantity(0.04 * units.meter**3)
+    body_density = Quantity(2600 * units.kilogram / units.meter**3)
+    liquid_density = Quantity(1000 * units.kilogram / units.meter**3)
+    Args = namedtuple("Args", ["body_volume", "body_density", "liquid_density"])
+    return Args(
+        body_volume=body_volume,
+        body_density=body_density,
+        liquid_density=liquid_density
+               )
+
+
+def test_basic_volume(test_args):
+    result = submerged.calculate_volume(test_args.body_volume, test_args.body_density, test_args.liquid_density)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.volume)
+    result_volume = convert_to(result, units.meter**3).evalf(5)
+    assert result_volume == approx(0.104, 0.001)

```

Comment:
It seems the submerged volume is higher than the full body volume. How is it possible?